src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -592,23 +599,23 @@ export class ActivityFeedItem extends React.Component<Props> {
           data = {
             ...data,
             subtext: t('label.outgoingSablierStream'),
-            itemValue: `- ${formattedAmount} ${symbol}`,
-            fullItemValue: `- ${formattedAmount} ${symbol}`,
+            itemValue: getValueWithSymbol(formattedAmount, symbol, false, false),
```

Comment:
Just a thought: wouldn't it be nicer to introduce functions like `getPositiveValueWithSymbol(amount, symbol)`? It'd be more readable than `getValueWithSymbol(formattedAmount, symbol, false, false)`, this way I have to wonder "what does this false, false mean?".

Reply:
I was going to mention the same thing, except with the alternative method of using an object to pass the options - so the parameter types would change from `(value: string, token: string, isPositive: boolean, doNotFormat: boolean)` to `(value: string, token: string, options: { isPositive: boolean, doNotFormat: boolean })`. This way one could still use conditions (like `formattedValue === '0'` in EventDetails).

Reply:
Yeah, good idea! Will update it shortly :) Thanks!

Reply:
Renamed it too, to make it more readable :)